x-pack/platform/packages/shared/kbn-streams-schema/src/models/group/base.ts

```diff
@@ -10,35 +10,37 @@ import { NonEmptyString } from '@kbn/zod-helpers';
 import { StreamDefinitionBase } from '../base';
 
 interface GroupBase {
-  description?: string;
   members: string[];
 }
 
 const groupBaseSchema: z.Schema<GroupBase> = z.object({
-  description: z.optional(z.string()),
   members: z.array(NonEmptyString),
 });
 
 interface GroupStreamDefinitionBase {
   group: GroupBase;
+  description: string;
 }
 
-const groupStreamDefinitionBaseSchema: z.Schema<GroupStreamDefinitionBase> = z.object({
+const groupStreamDefinitionSchemaBase: z.Schema<GroupStreamDefinitionBase> = z.object({
```

Comment:
should we use `z.intersection([ streamDefinitionSchemaBase, z.object({ group: groupBaseSchema }) ])`?

Reply:
`streamDefinitionSchemaBase` would include `name` i suppose? but `groupStreamDefinitionSchemaBase` (base) is used by `groupStreamUpsertRequestSchema` (api), and having `name` here would break this api.

Reply:
right, groupStreamUpsertRequestSchema shouldn't be using groupStreamDefinitionSchemaBase, or when it does, it should omit the `name` of `groupStreamDefinitionSchemaBase`. Unfortunately there's no `z.omit()`, just `z.object({}).omit()` I think.

Reply:
Maybe `StreamDefinitionBase` should not have `name:string`... but it's a big change. I think we can keep it simple for now. But let's focus on sig events first for a few days, and then get back to this PR, WDYT?

Reply:
Good with me to postpone this